lib/services/northBound/deviceGroupAdministrationServer.js

```diff
@@ -306,6 +324,34 @@ function clear(callback) {
     callback();
 }
 
+function renameObjetKeys(keysMap, obj) {
+    return Object.keys(obj).reduce((acc, key) => {
+        // Si la clave existe en el mapa, usar la clave renombrada
+        const renamedKey = keysMap[key] || key;
+
+        // Agregar la clave renombrada al acumulador
+        acc[renamedKey] = obj[key];
+
+        return acc;
```

Comment:
Maybe I'm not fully understanding this function but... I see how the renamed key is added, but in which moment the old key is removed?

Reply:
because its never added.  The full object is copied, and the copy returned.

Reply:
Fixed in https://github.com/telefonicaid/iotagent-node-lib/pull/1648/commits/936d78cc08e107b11275866519e221f88f134b6f